### ISSUE TITLE
Is the unified_position_quantization option still being worked on?

### ISSUE THREAD

**Issue by DeadlyMagikarps** [google/draco#916]
Is this feature still being worked on? It seems this option only works with the nodejs version with gltf-pipeline by setting `--draco.unifiedQuantization true`, but it's not available from here yet.

**Comment by ondys** [comment]
Yes this is still something we are working on for the C++ `draco_transcoder`
